### ISSUE TITLE
issues/comments

### ISSUE THREAD

**Issue by jsbain** [dgelessus/fistutils#2]
To get pip working at all, I had to add the line

```
    sys.stdout.isatty=lambda:False
```

since StdOutCatcher does not properly implement this.  

install docutils gives

No files/directories in /private/var/mobile/Containers/Data/Application/0AF05E1C-7F2F-401F-8730-DA27D9141558/tmp/pip-build-rb0tvf/docutils/pip-egg-info (from PKG-INFO)

**Comment by dgelessus** [comment]
Right, I put a few lines in my `pythonista_startup` to add `fileno` and `isatty` to the standard streams, a few other libraries require this too.

The second error happens with some packages (I noticed it with `hachoir-core`) though I'm not sure why exactly it happens. When I disable the automatic deletion of temporary files and check the folder mentioned in the error message, there are indeed no files in it.
